quic/s2n-quic-crypto/src/aead.rs

```diff
@@ -18,3 +19,89 @@ pub trait Aead {
         tag: &Self::Tag,
     ) -> Result;
 }
+
+impl Aead for LessSafeKey {
```

Comment:
why did this need to move?

Reply:
It was in `quic/s2n-quic-crypto/src/aesgcm/ring.rs`, which is now deleted.